catalog/values.yaml

```
@@ -126,10 +126,10 @@ catalog:
       value: 'world'
     client_identity:
       type: value
-      value: 'CN=gm-control,OU=Engineering,O=Decipher Technology Studios,=Alexandria,=Virginia,C=US'
+      value: 'C=US,ST=Virginia,L=Alexandria,O=Decipher Technology Studios,OU=Engineering,CN=*.greymatter.svc.cluster.local'
```

This should really be `CN=gm-catalog,OU=Engineering,O=Decipher Technology Studios,=Alexandria,=Virginia,C=US`.  
```suggestion
      value: 'CN=gm-catalog,OU=Engineering,O=Decipher Technology Studios,=Alexandria,=Virginia,C=US'
```

---

At least I believe it should, please test with this value.

---

Will test with your suggestions. What I set it to again is what `release-2.1` is doing right now.

---

Crashes with your suggestions. :(

---

Seems like it can connect but then doesn't have permissions with data.

---

Please post the logs.